crates/wasi-common/Cargo.toml

```diff
@@ -93,6 +93,11 @@ tokio = [
 ]
 exit = [ "wasmtime", "dep:libc" ]
 
+[package.metadata.docs.rs]
+features = ["trace_log", "wiggle_metadata", "wasmtime", "sync", "tokio", "exit"]
```

Comment:
Mind replacing this with `all-features = true`?

Reply:
Done